cmd/chisel/cmd_cut.go

```diff
@@ -82,29 +82,72 @@ func (cmd *cmdCut) Execute(args []string) error {
 		return err
 	}
 
+	archives, err := openArchives(release, cmd.Arch)
+	if err != nil {
+		return err
+	}
+
+	report, err := slicer.Run(&slicer.RunOptions{
+		Selection:   selection,
+		PkgArchives: archives,
+		TargetDir:   cmd.RootDir,
+	})
+	if err != nil {
+		return err
+	}
+
+	manifestSlices := locateManifests(selection.Slices)
+	if len(manifestSlices) > 0 {
+		pkgInfo, err := gatherPackageInfo(selection, archives)
+		if err != nil {
+			return err
+		}
+		_, err = generateDB(&generateDBOptions{
+			RootDir:        cmd.RootDir,
+			ManifestSlices: manifestSlices,
+			PackageInfo:    pkgInfo,
+			Slices:         selection.Slices,
+			Report:         report,
+		})
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// openArchives opens the archives listed in the release and returns the
+// archives mapped by package names.
+func openArchives(release *setup.Release, arch string) (map[string]archive.Archive, error) {
```

Comment:
Agreed. What do you think about `openPkgArchives`?

Reply:
The problem is the definition of `open` in this case, what does it mean? IMO all we are doing is checking the the package indeed exists in the archive and then setting that as its archive, which is why I though of "selecting". Incidentally, when we add priorities we are going to be "selecting" among archives for each package.

Reply:
I see your point but I think we should name it "select..." because it is the main intent of the function, the association between pkg and archive. Incidentally, this will also look for priorities and fetch once we land FIPS.

But at the same time I agree with you somewhat, so maybe it is best to split it into `openArchives` and `selectPkgArchives` and call both of them from the main function. Thoughts?

Reply:
Ya I agree. In that case, I would go with splitting them into `openArchives` and `selectPkgArchives`.

Reply:
Done in f2cebe3785ca9030b3b1a3250d4b60ad6992379e.